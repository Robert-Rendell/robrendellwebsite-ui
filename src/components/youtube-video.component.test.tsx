import React from "react";
import { render } from "@testing-library/react";
import { YouTubeVideoComponent } from "./youtube-video.component";

describe("YouTubeVideoComponent", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<YouTubeVideoComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
