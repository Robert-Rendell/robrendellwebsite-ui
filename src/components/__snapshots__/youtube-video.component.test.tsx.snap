// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YouTubeVideoComponent should match snapshot 1`] = `
<DocumentFragment>
  <iframe
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
    allowfullscreen=""
    height="315"
    title="YouTube video player"
    width="100%"
  />
</DocumentFragment>
`;
